Flabbergasted-Flamingos/Bangazon-admin-app
# Jcedittraining

# Description
Added the detailed view of training and added the ability to update training programs

## Number of Fixes
All of them

## Related Ticket(s)
Close #10 

## Steps to Test Solution

1. git fetch --all
2. git checkout JCedittraining
3. click on trainings on navbar
4. click on the training and see if it goes to the detailed view
5. once in the detailed view click on update and see if it goes to the update view
6. make sure you can update the training
